android/engine/src/main/java/org/smartregister/fhircore/engine/rulesengine/RulesFactory.kt

```diff
@@ -337,12 +337,21 @@ constructor(
     fun formatDate(date: Date, expectedFormat: String = "E, MMM dd yyyy"): String {
       return date.formatDate(expectedFormat)
     }
+
+    fun generateRandomIntLengthSix(): Int {
```

Comment:
I think this name is clearer, can you also please add a doc string

```suggestion
    fun generateRandomSixDigitInt(): Int {
```